index.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express')
+const teams = require('./teams.json')
+
+const app = express()
+
+app.get('./teams', (request, response) => {
+	response.send(teams)
+})
+
+app.get('/teams/:x', (request, response) => {
+	const x = request.params.x
+
+	const matchingTeams = teams.filter(() => {
+		return teams.id === number(x)
```

Comment:
This will throw an error. `number(x)` is not a thing but `Number(x)` is.  Also you're missing the handling of abbreviations here.